packages/calypso-build/postcss.config.js

```diff
@@ -1,7 +1,9 @@
-module.exports = ( { options: { preserveCssCustomProperties = true } } ) => ( {
+module.exports = ( {
+	options: { preserveCssCustomProperties = true, importCssCustomPropertiesFrom },
+} ) => ( {
 	plugins: {
 		'postcss-custom-properties': {
```

Comment:
could `postcss-custom-properties` plugin be left out if `importCssCustomPropertiesFrom` is falsey? That would effectively make this opt-in.